backend/src/main/java/com/woowacourse/gongcheck/domain/member/Member.java

```diff
@@ -42,6 +43,16 @@ public Member(Long id, String spacePassword, LocalDateTime createdAt, LocalDateT
         this.updatedAt = updatedAt;
     }
 
+    public void checkPassword(String password) {
+        if (!isSameSpacePassword(password)) {
+            throw new UnauthorizedException("공간 비밀번호와 입력하신 비밀번호가 일치하지 않습니다.");
+        }
+    }
+
+    private boolean isSameSpacePassword(String password) {
```

Comment:
```suggestion
    public void checkPassword(final String password) {
        if (!isSameSpacePassword(password)) {
            throw new UnauthorizedException("공간 비밀번호와 입력하신 비밀번호가 일치하지 않습니다.");
        }
    }

    private boolean isSameSpacePassword(final String password) {
```
인텔리제이에 자동생성시 매개변수에 final을 달 수 있는 옵션도 확인해보시면 좋을 것 같아요 :)